src/site/components/accordion.md-njk

```diff
@@ -13,7 +13,8 @@ The accordion component lets users show and hide sections of related content on
 
 {{ guidanceComponentPreview({
   componentName: "accordion",
-  exampleName: "default"
+  exampleName: "default",
+  macroOptions: macroOptions["accordion"]
```

Comment:
Adding this everywhere was far from ideal. Main alternative is to add `with context` to the import so that macro has the same context. Not sure what's the lesser of the two evils....